pyflakes/messages.py

```diff
@@ -116,6 +116,22 @@ def __init__(self, filename, loc, name):
         self.message_args = (name,)
 
 
+class DuplicateDictionaryKey(Message):
+    message = 'dictionary key %s repeated with different values'
+
+    def __init__(self, filename, loc, key):
+        Message.__init__(self, filename, loc)
+        self.message_args = (key,)
+
+
+class DuplicateVariableDictionaryKey(Message):
```

Comment:
I find this name to be confusing, as my brain reads it as being a "duplicate variable something ..."; the words 'duplicate' and 'key' are too far removed in the class name, and it doesnt help that this is now used to report conflicting values for the same key.

MultipleValueRepeatedLiteralKey and 
MultipleValueRepeatedNameKey ?

Im wondering whether we need two message classes; the word 'variable' could be added to the outputted message by VariableKey.__str__.